src/main/java/net/dv8tion/jda/api/EmbedBuilder.java

```diff
@@ -149,6 +130,55 @@ public EmbedBuilder clear()
         return this;
     }
 
+    /**
+     * Resets this builder to the state of the given builder.
+     * <br>All the parts of the given builder will be applied to this one.
+     *
+     * @param  builder
+     *         the existing builder
+     */
+    public void reset(EmbedBuilder builder)
```

Comment:
Maybe add Nullable annotation for consistency

I also think that reset is not the right method name, as it gives the impression that it resets all the builders data back to nothing, but this is not case because it applies the data of the given builder to this one